cli/src/bin/instructions/process_close_claim_status.rs

```diff
@@ -1,12 +1,22 @@
+use std::mem::transmute;
+
+use anchor_client::solana_client::rpc_filter::{Memcmp, RpcFilterType};
 use merkle_distributor::state::claim_status::ClaimStatus;
-use solana_sdk::compute_budget::ComputeBudgetInstruction;
 
 use crate::*;
 
 pub fn process_close_claim_status(args: &Args) {
     let program = args.get_program_client();
-
-    let claim_status_accounts: Vec<(Pubkey, ClaimStatus)> = program.accounts(vec![]).unwrap();
+    let true_bytes = unsafe { transmute::<bool, u8>(true) };
+    let claim_status_accounts: Vec<(Pubkey, ClaimStatus)> = program
+        .accounts(vec![
+            RpcFilterType::DataSize((8 + ClaimStatus::LEN) as u64),
+            RpcFilterType::Memcmp(Memcmp::new_raw_bytes(
+                8 + 32 + 8 + 8,
```

Comment:
Shouldn't it be
8 + 32 + 8 + 8 + 8 ?
disc, claimant:Pubkey, locked_amount: u64, locked_amount_withdrawn: u64, unlocked_amount: u6

Reply:
Fixed